2022-08-august/README.md

```diff
@@ -77,23 +77,34 @@ Feel free to reach out to anyone here on Slack, and post updates about your own
 
 Yes, I'm talking about Assemble. Or rather, Ian (14) and Sam (17) are:
 
-### TODO: Insert some catchy title for the hackathons grant>
+### 💸 Grants!!! For your hackathon!!! 💸
+![You get $500, Oprah meme](https://cloud-ncs2ypbyu-hack-club-bot.vercel.app/06qcy84.jpeg)
 
 
-### Fun things and trends
+At Assemble, <img src="https://ca.slack-edge.com/T0266FRGM-U0266FRGP-7f1bd1a815ac-512" width="16"> [Zach](https://zachlatta.com/) announced that [Hack Club and FIRST are offering a **✨ $500 grant ✨**](https://hackclub.com/hackathons/) to in-person high-school hackathons hosted before the end of 2022!!! All you need for the grant is a high-school team, a venue, and a custom website, and you get free tools and support too. Learn more at [hackclub.com/hackathons](https://hackclub.com/hackathons) and [APPLY HERE!](https://airtable.com/shrpYdJpAOphEWXan)
 
-TODO
 
-\
+
+### Things to do while procrastinating on math homework:
+- Add your own Orpheus drawing (there are HUNDREDS!!): [github.com/hackclub/dinosaurs](https://github.com/hackclub/dinosaurs)
+- Grab some popcorn 🍿 to join [#movie-nights](https://hackclub.slack.com/archives/C025QEVNRPW), find a quick laugh at [#memes](https://hackclub.slack.com/archives/CB19EP4MC), explore the Hack Club Slack [#hall-of-fame](https://hackclub.slack.com/archives/C028VGT0JMQ), or meet other Hack Clubbers on [#hack-night](https://hackclub.com/night/)
```

Comment:
Let's put hack night first, include "via zoom" and describe hall of fame better: "explore the best messages of the hack club slack" (and link to hackclub.com/slack)

Apologize for the improperly formatted message, I'm typing this on a phone lol

Reply:
Let's also add emojis for each channel as discussed here: https://hackclub.slack.com/archives/C01FZCM8GN4/p1660869953104479?thread_ts=1660869551.093119&channel=C01FZCM8GN4&message_ts=1660869953.104479

Reply:
no worries, you got it!